brainscore_vision/model_helpers/activations/temporal/core/__init__.py

```diff
@@ -0,0 +1,227 @@
+import copy
```

Comment:
let's add a note that this effectively duplicates the existing non-temporal activations extractor with new functionality for temporal models. And that we hope the two can be unified again in the future or that this new wrapper will supersede the previous one